domains/adri.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "adripo",
+    "email": "26493496+adripo@users.noreply.github.com"
```

Comment:
this should be an email we can email you with.

Reply:
I changed it